controllers/util/condition.go

```diff
@@ -0,0 +1,38 @@
+package util
```

Comment:
Condition related functions like `SetStatusCondition()` are implemented in k8s v1.19. We probably won't need this file, if we update k8s version in `go.mod`.

It's required to update go.mod to apply client-go and machinery version >= v0.19.0. The most current stable version is v0.19.7.

Reply:
Done. I will delete this file.